docs/installation.rst

```diff
@@ -67,15 +68,15 @@ Installing Sorcha in Development Mode
 
 Testing the Sorcha Installation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-You can check that the surveySimPP installation was done correctly, by running::
+You can check that the Sorcha installation was done correctly, by downloading the Sorcha source code as described in the  by running::
```

Comment:
"by downloading the Sorcha souce code as described <somewhere?> and then running:"